backend/tests/test_basics.py

```diff
@@ -3,10 +3,12 @@
 from app import create_app
 
 class BasicsTestCase(unittest.TestCase):
+    # TODO Create a test database to fill with dummy data
```

Comment:
We do have a test Airtable, the task @celinehuang is working on will allow for different app configs so once that's merged in you can implement this